src/VisualStudio/VisualStudioDiagnosticsToolWindow/OptionPages/InternalOptionsControl.cs

```diff
@@ -145,21 +145,23 @@ private UIElement CreateControl(IOption option, string languageName = null, stri
         {
             if (option.Type == typeof(bool))
             {
-                var checkBox = new CheckBox() { Content = option.Name + GetLanguage(languageName) + GetAdditionalText(additional) };
+                var checkBox = new CheckBox() { Content = option.Name.Replace('_', ' ') + GetLanguage(languageName) + GetAdditionalText(additional) };
                 BindToCheckBox(checkBox, option, languageName);
                 return checkBox;
             }
 
             if (option.Type == typeof(int))
             {
-                var label = new Label() { Content = option.Name + GetLanguage(languageName) + GetAdditionalText(additional) };
+                var label = new Label() { Content = option.Name.Replace('_', ' ') + GetLanguage(languageName) + GetAdditionalText(additional) };
                 var textBox = new TextBox();
                 BindToTextBox(textBox, option, languageName);
 
                 var panel = new StackPanel();
                 panel.Children.Add(label);
                 panel.Children.Add(textBox);
 
+                AddSearchHandler(label);
```

Comment:
A small suggestion (doesn't need to be in this PR):

Is it possible to do a change similar to the following:


```csharp
// renamed the abstract CreateOptionPage to CreateOptionPageCore, this is what needs to be overridden now.
protected abstract AbstractOptionPageControl CreateOptionPageCore(IServiceProvider serviceProvider, OptionStore optionStore);


protected AbstractOptionPageControl CreateOptionPage(IServiceProvider serviceProvider, OptionStore optionStore)
{
    CreateOptionPageCore();
    // loop over all labels, comboboxes, checkboxes and call AddSearchHandler
}
```

Reply:
If this were winforms, I'd absolutely do that.. but its not, and I don't think WPF has a convenient way of iterating child controls, across the myriad of options for containers and stack layouts and grids etc.

Plus, 99% of the search handler setup is already automatic because most things just call the base `Bind...` method anyway, so I don't think this would save much.